controllers/bootstrap/init.go

```diff
@@ -77,6 +78,30 @@ func InitResources(mgr manager.Manager) error {
 		return err
 	}
 
+	// Checking existing CommonService subscription
```

Comment:
if ns is not in openshift-operators or ibm-common-services {
  create cs subscription
  return
}